lib/Config.js

```diff
@@ -13,14 +13,11 @@ const { isValidBucketName } = s3routes.routesUtils;
 const validateAuthConfig = arsenalAuth.inMemory.validateAuthConfig;
 const { buildAuthDataAccount } = require('./auth/in_memory/builder');
 const validExternalBackends = require('../constants').externalBackends;
-const {
-    azureAccountNameRegex,
-    base64Regex,
-    allowedUtapiEventFilterFields,
-    allowedUtapiEventFilterStates,
-    supportedLifecycleRules,
+const { azureAccountNameRegex, base64Regex,
+    allowedUtapiEventFilterFields, allowedUtapiEventFilterStates,
+    supportedLifecycleRules
 } = require('../constants');
-const { utapiVersion } = require('utapi');
+// const { utapiVersion } = require('utapi');
```

Comment:
"mock" the variable definition, so you don't need to add `eslint-disable-line no-undef`

```suggestion
const utapiVersion = 0; // require('utapi');
```